modules/platforms/cpp/tests/fake_server/fake_server.cpp

```diff
@@ -0,0 +1,5 @@
+//
+// Created by Ed on 21.11.2025.
+//
+
+#include "fake_server.h"
```

Comment:
Empty file, let's remove.

Reply:
Not anymore. I moved big functions to .cpp file.